app_store_connect_api.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "faraday"
+  spec.add_dependency "faraday-retry"
```

Comment:
Doesn't Faraday contain a retry middleware by default? (Sorry for butting in. 😀 )